config/routes.rb

```diff
@@ -320,6 +320,7 @@ def scoped_navigation_routes(context, navigation)
       get "/questions/use-gyr", to: "ctc/questions/use_gyr#edit", as: :questions_use_gyr
       get "/questions/returning-client", to: "ctc/questions/returning_client#edit", as: :questions_returning_client
       get "/already-filed", to: "ctc/offboarding/already_filed#show", as: :offboarding_already_filed
+      get "/cant-use-getctc", to: "ctc/offboarding/puerto_rico_sign_up#show", as: :offboarding_cant_use_getctc
```

Comment:
its weird to have this url be so generic when it is actually very specific to puerto rico. Is it possible that we can do `/cant-use-getctc-pr` instead?

Reply:
agree that it's weird! i was just using the one specified in the story, but can ask yvonne if it's ok to change it